CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt

```diff
@@ -564,36 +567,36 @@ Every target revealed by Sense Heartbeat that is communicated to an ally and is
 then defeated grants the Doc 20 adrenaline points. This ability may be 
 sustained for 5 adrenaline per turn as a free action.
 
-----Mild Steroid	-----
+==Mild Steroid ==
 Adrenaline Reward:	15
 Range:			Touch
 Action:			1 Action
 Effect:			The patient deals an extra 2*(patient's strength) damage with
-melee weapons, has -8% to miss with melee weapons, and moves +1m per action. As
+melee weapons, has -1 to miss with melee weapons, and moves +1m per action. As
 a side effect, the target takes a -1 to both Intelligence and charisma for the
 duration of the effect. This trick only works on biologic patients. This effect
 lasts for 4 turns.
 
-----Preventative Medicine	-----
+==Preventative Medicine ==
 Adrenaline Reward:	20
 Range:			Touch
 Action:			1 Action
 Effect:			Target gets a 30 point capacitive shield for the duration of 
 the engagement. This trick works on any race of ally. 
 
-----Nanite Restore	-----
+==Nanite Restore ==
 Adrenaline Reward:	n/4
 Range:			Touch
 Action:			Free Action
 Effect:			Transfer n number of Adrenaline points from the Doc to the 
 patient. If This trick is used on another doctor, no adrenaline points are earned. 
```

Comment:
>Transfer n number of Adrenaline points from the Doc to the 
  patient. If This trick is used on another doctor, no adrenaline points are earned. 

The name doesn't up with the effect, anymore. Nanite Restore looks more like "Adrenaline Restore," and it doesn't make any sense to transfer adrenaline points to the patient if the patient isn't a doctor.